### PR TITLE
Pattern: Writer Archive Template

### PR DESCRIPTION
**Pull request by carolinan**
**Description**

Adds a full page pattern for the writer archive template.

For https://github.com/WordPress/twentytwentyfour/issues/351
Figma: https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?node-id=492%3A1303&mode=dev

**Screenshots**
64 local__cat=1 (1)

**Testing Instructions**
Delete all blocks in archive.html, and add
`<!-- wp:pattern {"slug":"twentytwentyfour/archive-writer"} /-->`
View an archive page.

### CODE REVIEW

**Comment by carolinan** [comment]
Note that if you insert the pattern via the site editor, the header and footer will not load.

File: theme.json [hunk 1]
```diff
@@ -650,6 +650,9 @@
 					"lineHeight": "1.5"
 				}
 			},
+			"core/query-title": {
+				"css": "& span {font-style: italic;}"
```
Comment (MaggieCabrera) [inline theme.json:654]: @carolinan what do you think about this?

**Comment by MaggieCabrera** [comment]
I made a few changes so it looks closer to the design

File: theme.json (hunk 1 above)
Comment (carolinan) [reply theme.json:654]: It is the only way to style the query object name (like category name). :)

Comment (MaggieCabrera) [reply theme.json:654]: yeah, I just feel like it looks so much better like this

**Comment by MaggieCabrera** [comment]
@carolinan if all is good with you I'd like to merge this one in

**Review by MaggieCabrera — APPROVED** [review]

**Pull request merged**